src/FakeItEasy.Analyzer/VisualBasic/Resources.resx

```diff
@@ -103,4 +103,10 @@
     <data name="ArgumentConstraintNullabilityMismatchMessageFormat" xml:space="preserve">
     <value>Parameter '{0}' of method '{1}' is nullable, but argument constraint '{2}' is not. Calls where this argument is Nothing will not be matched.</value>
   </data>
+  <data name="MakeConstraintNullableCodeFixTitle">
+    <value>Change constraint type to nullable</value>
+  </data>
+  <data name="MakeNotNullConstraintCodeFixTitle">
+    <value>Change to 'not Nothing' constraint</value>

```

Comment:
I'm sure you did this intentionally, but we'll still end up with a constraint that says `IsNotNull`.
I worry about confusing the VB developers.
Is this a situation where we're better off using the actual method name rather than an English description of the outcome? Maybe both?